### DISCUSSION TITLE
https://server.siliconcompiler.com/ doesn't resolve

### DISCUSSION THREAD

**Discussion by stringinator** [siliconcompiler/siliconcompiler#787] [Development]
I tried to connect for the Beta cloud and https://server.siliconcompiler.com/ doesn't resolve in my Chrome browser. Is the web-based access up and running yet?

**Comment by stringinator** [comment]
Any chance this is related to the recent AWS downtime?
![image](https://user-images.githubusercontent.com/25567157/145084121-b1929675-b57d-4898-b162-74514363fe4e.png)

**Reply by nmoroze** [reply]
You shouldn't expect to see anything if you directly visit https://server.siliconcompiler.com in-browser. However, if you run `sc-configure` and enter the credentials you received via email after signing up for the beta, you can then run `sc-ping` to quickly check your connection!

**Reply by stringinator** [reply]
Thanks!

**Reply by WRansohoff** [reply]
Yep, Noah is correct - server.siliconcompiler.com is where the client sends its API requests, but we don't host any web pages there at the moment. Since you mention it, though, maybe we should consider putting a more explanatory 404 page at the root of that domain.

**Reply by raspiduino** [reply]
I just signed up an account today and after I configured it using `sc-configure`, `sc-ping` just returned `Error fetching user information from the remote server.` Is this service limited in some areas?

**Reply by WRansohoff** [reply]
Hm, sorry that the remote interface isn't working for you - I just tried running `sc-ping` and it seemed to work with a new account.

Could you share what the "address" field is in your ~/.sc/credentials file (or C:\Users\[your_user]\.sc\credentials on Windows)? It should be `https://server.siliconcompiler.com` or `server.siliconcompiler.com`, and the username/passwords should exactly match what you received in the welcome email.

**Reply by raspiduino** [reply]
Thanks for replying so fast! When I entered like the email - `https://server.siliconcompiler.com` didn't work, but `server.siliconcompiler.com` worked for me! I think you should change the server address in the email from `https://server.siliconcompiler.com` to `server.siliconcompiler.com` .

Thank you very much for saving my time!

P/S: Cloud saved my time. I tried to install tools from https://docs.siliconcompiler.com/en/latest/reference_manual/tools.html but the UHDM in surelog failed to build :(

**Reply by WRansohoff** [reply]
Ah, thank you for reporting this. The server requires TLS, so the client should be connecting over HTTPS either way, but I've updated our welcome email template to drop the 'https://' prefix for now. Would you mind sharing which OS and Python version you are using?

I'm glad that you found the cloud interface useful! We do also have install scripts for the core EDA tools [in the `setup/` directory](https://github.com/siliconcompiler/siliconcompiler/tree/main/setup) if you want to try installing Surelog that way, but those setup scripts primarily target Ubuntu 20.04.

**Reply by raspiduino** [reply]
Thanks! I'm running Debian Buster (Linux 5.10.68) with Python 3.7.3 and gcc 8.3.0. If you need more information about the environment, I will happily provide them!

Edit: Oh, there is Docker! I will try it tomorrow. Thanks again!
